packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -58,28 +64,166 @@ class AllocationTableViewState extends State<AllocationTableView>
         controller.computeAllLibraries(rebuild: true);
       });
     });
+
+    addAutoDisposeListener(controller.updateClassStackTraces, () {
+      setState(() {});
+    });
+
+    addAutoDisposeListener(controller.treeChangedNotifier, () {
+      if (controller.isTreeChanged) {
```

Comment:
why does this fire if the tree hasn't changed?

Reply:
It only fires if the tree has changed true or false.  However, the tree never resets back to false once fired (value == true). So the test is meaningless - removed.